src/plugins/embeddable/public/__stories__/embeddable_panel.stories.tsx

```diff
@@ -128,7 +128,7 @@ Default.args = {
   hideHeader: false,
   loading: false,
   showShadow: false,
-  showBorder: true,
+  showBorder: false,
```

Comment:
Why the change to `false` here? 🤔

Reply:
There are no more cases where we want panels without borders, e.g. in the solutions, in canvas, dashboard without margins, than panels with borders, e.g. dashboard with margins. I figured the default might as well be no border